spec/controllers/people_controller_spec.rb

```diff
@@ -0,0 +1,128 @@
+# spec/controllers/people_controller_spec.rb
+require 'rails_helper'
+
+RSpec.describe PeopleController, type: :controller do
+  describe "GET #index" do
+    subject { get :index }
+    before(:each) { @response = subject } 
+
+    it "renders the template" do
+      expect(@response).to render_template(:index)
+    end
+    it "loads all of the people into @people" do
+      expect(@response.body.length).to be_truthy
+    end
+  end
+
+  describe "GET #show" do
+    before(:each) { @person = create(:person) }
```

Comment:
Instead of before blocks you can utilize RSpec's `let`s.

```
let(:person) { FactoryGirl.create :person }
```

Generally we also prefer putting `FactoryGirl` in front of `.create` (or `.build`) so that we're being explicit in what is doing the creating (just like you've done `RSpec.describe` at the top of this file instead of `describe`).